docs/source/Plugin/_plugin_substitutions_p12x.repl

```diff
@@ -0,0 +1,12 @@
+.. |P121_name| replace:: :cyan:`HCM5883L`
```

Comment:
I think this file also needs to be included in `_plugin_substitutions.repl` (in the same folder)

Reply:
I was hoping this is already done by the maintainer, since it is global.

Reply:
Well, _you_ added that _12x.repl file (and also did I, in another PR 😛), and Sphinx gets quite grumpy about missing files.